### PR TITLE
- correct isolangs.json (Traditional and Simplified Chinese)

### PR DESCRIPTION
**Pull request by dirkarnez**
Please consider update isolangs.json if it works, thank you!

### CODE REVIEW

**Comment by avm99963** [comment]
Hi, and thank you so much for this pull request! Sorry for taking so long to reply, I've been very absent from this project until now and didn't see the PR until recently.

When I saw this PR, I went to check whether there were also some other mistakes in the isoLangs variable definition, and I saw that there were a handful of languages which were also set incorrectly (some, for instance, had the wrong language code).

Therefore, I've added a tool which automatically gets all possible targets from the Google Translate servers and generates automatically the isoLangs variable, by merging the preexistent `isoLangs.json` file (which I've slightly modified) into the list of target languages: https://gerrit.avm99963.com/c/translateselectedtext/+/870.

This is the change which uses the tool to modify the isoLangs variable: https://gerrit.avm99963.com/c/translateselectedtext/+/871

In conclusion, this issue is now solved. Thank you so much for opening this pull request and bringing this issue to my attention! I'm sorry that I didn't merge the PR in the end, but if you want to further contribute in the project, either by filling bug reports or sending pull requests, please feel welcome to do so! I'm now trying to bring this project to a better state by rewriting the code in a much better way (I've learned a lot since I started to build this extension!), which is something that I should have already done a long time ago.
